airbyte-cdk/python/unit_tests/sources/declarative/transformations/test_remove_fields.py

```diff
@@ -10,39 +10,64 @@
 
 
 @pytest.mark.parametrize(
-    ["input_record", "field_pointers", "expected"],
+    ["input_record", "field_pointers", "condition", "expected"],
     [
-        pytest.param({"k1": "v", "k2": "v"}, [["k1"]], {"k2": "v"}, id="remove a field that exists (flat dict)"),
-        pytest.param({"k1": "v", "k2": "v"}, [["k3"]], {"k1": "v", "k2": "v"}, id="remove a field that doesn't exist (flat dict)"),
-        pytest.param({"k1": "v", "k2": "v"}, [["k1"], ["k2"]], {}, id="remove multiple fields that exist (flat dict)"),
+        pytest.param({"k1": "v", "k2": "v"}, [["k1"]], None, {"k2": "v"}, id="remove a field that exists (flat dict)"),
```

Comment:
I mentioned it in another comment, but these tests should be updated to pass `""` instead of `None` as a condition. `condition` is not an Optional so it should always be set + we need to verify we get the desired behavior when passing an empty string since that's the default behavior

Reply:
changed `None` to `""`.

added some new cases to tests (and examples as well):

- `"{{ property == 'v1' }}"`
- `"{{ property|length > 5 }}"`
- `"{{ property is integer }}"`